Description: Explicitly link with gmodule
Author; Lionel Le Folgoc <mrpouit@ubuntu.com>
Bug: https://bugzilla.xfce.org/show_bug.cgi?id=8467

--- thunar-1.2.3.orig/configure.in
+++ thunar-1.2.3/configure.in
@@ -150,6 +150,7 @@ dnl ***********************************
 XDT_CHECK_PACKAGE([EXO], [exo-1], [0.6.0])
 XDT_CHECK_PACKAGE([GLIB], [glib-2.0], [2.18.0])
 XDT_CHECK_PACKAGE([GIO], [gio-2.0], [2.18.0])
+XDT_CHECK_PACKAGE([GMODULE], [gmodule-2.0], [2.10.0])
 XDT_CHECK_PACKAGE([GTHREAD], [gthread-2.0], [2.18.0])
 XDT_CHECK_PACKAGE([GTK], [gtk+-2.0], [2.14.0])
 XDT_CHECK_PACKAGE([GDK_PIXBUF], [gdk-pixbuf-2.0], [2.14.0])
--- thunar-1.2.3.orig/thunarx/Makefile.am
+++ thunar-1.2.3/thunarx/Makefile.am
@@ -49,6 +49,7 @@ libthunarx_2_la_SOURCES =						\
 libthunarx_2_la_CFLAGS =						\
 	$(GLIB_CFLAGS)							\
 	$(GIO_CFLAGS)							\
+	$(GMODULE_CFLAGS)						\
 	$(GTK_CFLAGS)							\
 	$(PLATFORM_CFLAGS)
 
@@ -61,6 +62,7 @@ libthunarx_2_la_LDFLAGS =						\
 libthunarx_2_la_LIBADD =						\
 	$(GLIB_LIBS)							\
 	$(GIO_LIBS)							\
+	$(GMODULE_LIBS)							\
 	$(GTK_LIBS)
 
 pkgconfigdir = $(libdir)/pkgconfig
